Lombiq.Tests.UI/OrchardCoreUITestBase.cs

```diff
@@ -145,6 +145,17 @@ public abstract class OrchardCoreUITestBase
     protected virtual Task ExecuteTestFromExistingDBAsync(
         Func<UITestContext, Task> testAsync,
         Browser browser,
+        Func<OrchardCoreUITestExecutorConfiguration, Task> changeConfigurationAsync = null) =>
+        ExecuteTestFromExistingDBAsync(testAsync, browser, customAppAssemblyPath: null, changeConfigurationAsync);
+
+    /// <summary>
+    /// Executes the given UI test, starting the app from an existing SQLite database available in the App_Data or in
+    /// the given folder.
+    /// </summary>
+    protected virtual Task ExecuteTestFromExistingDBAsync(
+        Func<UITestContext, Task> testAsync,
+        Browser browser,
+        string customAppAssemblyPath = null,
```

Comment:
If we're talking about a snapshot then it won't contain the assembly.

```suggestion
        string customAppFolderPath = null,
```